src/SmartComponents/OperatingSystems/OperatingSystemsCard.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { TemplateCard, TemplateCardBody, TemplateCardHeader } from '../../PresentationalComponents/Template/TemplateCard';
+import { PieChart } from '../../ChartTemplates/PieChart/PieChartTemplate';
+
+/**
+ * Operating systems card for showing the ratio of operating systems used.
+ */
+const OperatingSystemsCard = () => {
+    return <TemplateCard appName='OperatingSystems'>
+        <TemplateCardHeader subtitle='Operating systems'>
+        </TemplateCardHeader>
+        <TemplateCardBody>
+            <div style={ { width: '200px' } }>
+                <PieChart
+                    ariaDesc="Operating systems used"
+                    ariaTitle="Pie chart operating systems"
+                    constrainToVisibleArea={ true }
+                    data={ [
+                        { x: 'Red Hat Enterprise Linux 8', y: 20 },
```

Comment:
same as above abstract this data into a const, and auto build the legend off its values